java/client/src/main/java/glide/api/models/commands/bitmap/BitEncodingOption.java

```diff
@@ -0,0 +1,28 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.bitmap;
+
+import lombok.Getter;
+
+public class BitEncodingOption {
```

Comment:
If all these classes are used with `BitField` commands only - you can move it into `BitFieldOptions.java`